Bottom: 294c9b2fdbe6f12c8fbb926f38ecf426aa88bcef
Top:    ccfad21de78e9610d0484b35f373dc1414688aa7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:47:32 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index d387f43..0b03204 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
